README.md

```diff
@@ -14,92 +13,47 @@
-
-Find the answers to the most common questions in [our FAQ](https://www.octobot.info/usage/frequently-asked-questions-faq).
-
-## Your Octobot
-<a href="https://www.octobot.online/guides/#telegram"><img src="../assets/telegram-interface.png" height="414" alt="Telegram interface"></a>
-[![Twitter Interface](../assets/twitter-interface.png)](https://www.octobot.info/interfaces/twitter-interface)
-
-OctoBot is highly customizable using its configuration and tentacles system. 
-You can build your own bot using the infinite [configuration](https://www.octobot.online/guides/#trading_modes) possibilities such as 
-**technical analysis**, **social media processing** or even **external statistics management** like google trends.
-
-OctoBot is **AI ready**: Python being the main language for OctoBot, it's easy to integrate machine-learning libraries such as [Tensorflow](https://github.com/tensorflow/tensorflow) or
-any other lib and take advantage of all the available data and create a very powerful trading strategy. 
-
-Octobot's main feature is **evolution** : you can [install](https://www.octobot.info/advanced_usage/tentacle-manager), 
-[modify](https://developer.octobot.info/guides/customize-your-octobot) and even [create](https://developer.octobot.info/guides/developer-guide) any tentacle you want to build your ideal cryptocurrency trading robot. You can even share your OctoBot evolutions!
 
 ## Hardware requirements
 - CPU : 1 Core / 1GHz
 - RAM : 250 MB
 - Disk : 1 GB
 
-## Installation
-OctoBot's installation is **very simple**... because it is **very documented** ! See the [installation guides](https://www.octobot.online/guides/#installation) for more info.
-
-#### One click deployment
-Deploy your OctoBot on the OctoBot Cloud. No installation is required and your OctoBot will always be online and reachable.
-
-[![Telegram News](https://img.shields.io/static/v1?label=Deploy&message=now&color=007bff&style=for-the-badge)](https://octobot.click/gh-deploy)
-
-
-#### [With executable](https://www.octobot.info/installation/with-binary)
-Follow the [2 steps installation guide](https://www.octobot.online/local_installation/) 
-
-In short:
-- Use the latest release on the [release page](https://github.com/Drakkar-Software/OctoBot/releases/latest)
-
-#### [With Docker](https://www.octobot.info/installation/with-docker)
-Follow the [docker installation guide](https://www.octobot.online/local_installation/) 
-
-In short :
-```
-docker run -itd --name OctoBot -p 80:5001 -v $(pwd)/user:/octobot/user -v $(pwd)/tentacles:/octobot/tentacles -v $(pwd)/logs:/octobot/logs drakkarsoftware/octobot:stable
-```
-And then open [http://localhost](http://localhost).
+## Installation  
+OctoBot's installation is **very simple**, you can either:
+- [ Deploy your OctoBot on OctoBot Cloud](https://octobot.cloud/). On OctoBot Cloud, your robot will be running and reachable at all time, it will also benefit from cloud only features.
```

Comment:
```suggestion
- [Deploy your OctoBot on OctoBot Cloud](https://octobot.cloud/). On OctoBot Cloud, your robot will be running and reachable at all time, it will also benefit from cloud only features.
```